claims_hosp/delphi_claims_hosp/update_indicator.py

```diff
@@ -114,9 +114,7 @@ def geo_reindex(self, data):
         elif self.geo == "hrr":
             data_frame = data  # data is already adjusted in aggregation step above
         else:
-            logging.error(
-                "%s is invalid, pick one of 'county', 'state', 'msa', 'hrr', 'hhs', nation'",
-                self.geo)
+            self.logger.error("%s is invalid, pick one of 'county', 'state', 'msa', 'hrr', 'hhs', nation'", self.geo)
```

Comment:
you should remove the `import logging` at the top of the file too!
 
```suggestion
            self.logger.error("geo specified is invalid, pick one of 'county', 'state', 'msa', 'hrr', 'hhs', nation'", geo=self.geo)
```